STCore/src/commonMain/kotlin/com/infomaniak/multiplatform_swisstransfer/managers/UploadManager.kt

```diff
@@ -314,6 +316,30 @@ class UploadManager(
         uploadController.removeData()
     }
 
+    @Throws(
+        CancellationException::class,
+        EmailValidationException::class,
+        ApiException::class,
+        NetworkException::class,
+        UnexpectedApiErrorFormatException::class,
+        UnknownException::class,
+    )
+    suspend fun verifyEmailCode(code: String, email: String): AuthorEmailToken {
+        return uploadRepository.verifyEmailCode(VerifyEmailCodeBody(code, email))
+    }
+
+    @Throws(
+        CancellationException::class,
+        ApiException::class,
+        NetworkException::class,
+        UnexpectedApiErrorFormatException::class,
+        UnknownException::class,
+    )
+    suspend fun resendEmailCode(emailAddress: String) {
```

Comment:
`email` or `emailAddress` ? Because we have two names for the same things in `verifyEmailCode` and `resendEmailCode`.

Reply:
Since everything already mentions "email" why not keep "address" for both instead